pkgs/development/python-modules/codefind/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, poetry-core
+}:
+
+buildPythonPackage rec {
+  pname = "codefind";
+  version = "0.1.3";
+  format = "pyproject";
```

Looks like that only Python > 3.8 is [supported](https://github.com/breuleux/codefind/blob/master/pyproject.toml#L9).

```suggestion
  format = "pyproject";

  disabled = pythonOlder "3.8";
```